samcli/commands/local/invoke/cli.py

```diff
@@ -162,7 +162,7 @@ def do_cli(  # pylint: disable=R0914
 
             # Invoke the function
             context.local_lambda_runner.invoke(
-                context.function_name, event=event_data, stdout=context.stdout, stderr=context.stderr
+                context.function_identifier, event=event_data, stdout=context.stdout, stderr=context.stderr
```

Comment:
Is this the property that we are going to use from now on? And it will have the value like `Stack/FunctionName`?

Reply:
yes, function_identifier can be either function logical ID, function name or full path like Stack/FunctionLogicalId